lib/msal-core/test/UserAgentApplication.spec.ts

```diff
@@ -807,7 +807,7 @@ describe("UserAgentApplication.ts Class", function () {
 
         it("LoginStartPage is cached on acquireTokenRedirect call", (done) => {
             const tokenRequest: AuthenticationParameters = {
-                scopes: ["S1"], 
+                scopes: ["s1"], 
```

Comment:
Are these not getting set to lower case by `trimAndConvertToLowerCase`? Does that usually happen before `AuthenticationParameters` is created?

Reply:
I changed all scopes except the ones in the tests for trimAndConvertToLowerCase to lower case. I'll look over them again and see if there's any other tests where we can capitalize and assume they'll be downcased by trimAndConvert..

Reply:
Changed some of these back to confirm, they ARE being trimmed and converted in the scopes validation process before the `AuthenticationParameters are instantiated`.